app/controllers/users_controller.rb

```diff
@@ -1,10 +1,18 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: %i(new show create)
+  before_action :load_user, except: %i(index new create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = User.order(:name).page params[:page]
+  end
+
   def new
     @user = User.new
   end
 
   def show
```

Comment:
xóa hết nội dung trong method show, chuyển thành
```suggestion
  def show; end
```